README.md

```diff
@@ -103,6 +105,22 @@ The build and deploy documents for the API are located in the [api](./api) direc
 
 The build and deploy documents for the Web are located in the [web](./web) directory of this project.
 
+### READ-ONLY ADMIN
+
+To effectively assist teams calling 7-7000 for application support after hours, we created read-only access to the Project Registry so that they can look up who the Technical Lead for the app in trouble as sometimes the caller may not know this information. Read-only admin access will allow the user to view all of the projects that the Registry App manages but can't make any changes to it.
```

Comment:
To effectively assist teams calling 7-7000 for application support after hours, we created read-only access to the Project Registry. This allows looking up information for the app's Technical Lead as sometimes the caller may not have this.